primary/src/routes/scouting.ts

```diff
@@ -78,6 +79,14 @@ router.get('/match*', wrap(async (req, res) => {
 	let allianceLocale = (alliance.toLowerCase().startsWith('b')) ? req.msg('alliance.blueShort') : req.msg('alliance.redShort');
 	let title = `#${scoringdata[0]?.match_number} - ${teamKey.substring(3)} ${allianceLocale} | ${req.msg('scouting.match')}`;
 
+	// 2024-02-05, M.O'C: Add super-scout pit text to page
+	let pitFind: PitScouting[] = await utilities.find('pitscouting', { 'org_key': org_key, 'event_key' : eventKey, 'team_key' : teamKey }, {});
```

Comment:
In this case, you can just use utilities.findOne:

```ts
let pitFind = await utilities.findOne('pitscouting', { 'org_key': org_key, 'event_key' : eventKey, 'team_key' : teamKey }, {});
let pit_super_data: StringDict|null = null;
if (pitFind && pitFind.super_data) pit_super_data = pitFind.super_data);
```

There was a period where we had to manually annotate the return type of all utilities functions, but now the return type for find and findOne is automatically determined